x-pack/plugins/maps/public/connected_components/timeslider/timeslider.tsx

```diff
@@ -103,21 +127,29 @@ class KeyedTimeslider extends Component<Props, State> {
   };
 
   _onNext = () => {
+    let lastTick = this.state.max;
+    if (this.props.timeRangeStep !== 1) {
+      lastTick = this.state.ticks[this.state.ticks.length - 1].value;
+    }
     const from =
-      this._doesTimesliceCoverTimerange() || this.state.timeslice[1] === this.state.max
+      this._doesTimesliceCoverTimerange() || this.state.timeslice[1] === lastTick
         ? this.state.ticks[0].value
         : this.state.timeslice[1];
     const to = from + this.state.range;
-    this._onChange([from, to <= this.state.max ? to : this.state.max]);
+    this._onChange([from, to <= lastTick ? to : lastTick]);
```

Comment:
This change to `onNext`, to restrict the stepping to only the ticks introduces a different behavior as before. 

I'm wondering if we need to do this. This is related to the regression (2) here https://github.com/elastic/kibana/pull/121739#pullrequestreview-851886979.  It's now slightly different. While we have retained the padding here, the stepping behavior has changed. Now the last step is no longer the "padded"-tick at the end.

Reply:
My thoughts here was to prevent navigating to "padded" tick at the start/end in case of selected time period, but not in Auto. Basically, if the user chose, let's say, "1 day", why should he be able to go between last tick and the end of the timeline, which does not fit to the "1 day" option?

Reply:
The padded tick at start/end should be reachable when using next/prev buttons regardless of the selected interval. This behavior should not change from this PR